src/styles/App.css

```diff
@@ -1,5 +1,7 @@
 body {
```

Comment:
Also, is the home.css still being used? Otherwise, it could be deleted or out commented (in case we are going to need it again)

Reply:
we will delete during refactoring👍🏻👍🏻